### DISCUSSION TITLE
Found duplicate events (reload & reopen)

### DISCUSSION THREAD

**Discussion by ghost** [gdh1995/vimium-c#727] [Q&A] [answered]
Navigating page > `r | Reload current frame` & Navigating History > ` <a-s-r> | Reopen current page` both does the same thing. Reloading the existing page. I'm quite not sure I've understood how both differ to each other. Caz the default bindings for those do the same thing i.e reloading in the existing tab. Is the latter even needed ?

**Comment by gdh1995** [comment, marked as answer]
`Reopen` means to abort navigation history, so that one can get a "somehow clean" page state.
